conda/plugins/manager.py

```diff
@@ -159,6 +161,24 @@ def get_solver_backend(self, name: str = None) -> type[Solver]:
 
         return backend
 
+    def get_pre_command_hook_actions(
+        self, command: str, args: Namespace
+    ) -> Iterable[Callable]:
+        """
+        Returns the ``CondaPreCommand.action`` functions registered by the ``conda_pre_commands``
+        hook. We use ``functools.partial`` to preconfigure these functions with the arguments
+        they need.
+
+        :param command: name of the command that is currently being invoked
+        :param args: arguments of the running command
+        """
+        # Load pre-commands available to run
+        pre_command_hooks = self.get_hook_results("pre_commands")
+
+        for pre_command in pre_command_hooks:
+            if command in pre_command.run_for:
+                yield functools.partial(pre_command.action, command, args)
```

Comment:
Does the partial give us anything here compared to just passing command and args? Partials are annoying to debug since they have the parameters baked into them, while passing the variables doesn't take up more memory and enables to show up as local variables during debugging.